src/components/PanelV2/__tests__/PanelV2.spec.js

```diff
@@ -3,38 +3,103 @@
  * @copyright IBM Security 2019
  */
 
-import { mount } from 'enzyme';
-import React, { Fragment } from 'react';
-
-import { label, onClick } from '../_mocks_';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
 
 import { Button, PanelV2, PanelContent } from '../../..';
 
 describe('PanelV2', () => {
-  it('renders', () => {
-    const wrapper = mount(
+  test('should have no Axe or DAP violations', async () => {
```

Comment:
I'm curious about the choice of `test` versus `it` here?

Reply:
😆 I guess it go in my head while reading the React testing library docs & see all of their examples in that format. Though honestly I do kinda like `test` or `it` here, especially because I've flattened out the tests a bit. But do you have a preference?

(One more note actually -- it make help to do new structure with `test` to help us differentiate right away between new/re-evaluated tests and old tests?)

Reply:
I've a feeling `it` is just an alias for familiarity between other testing libraries. I like that idea about differentiating!